### PR TITLE
[FEATURE] add replacer for jQuery.fn.control

### PR DESCRIPTION
**Pull request by stopcoder**
`jQuery.fn.control` is now able to be replaced by `UI5Element.closestTo`

### CODE REVIEW

File: src/tasks/helpers/finders/JQueryControlCallFinder.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		node.name.includes("$") || node.name.toLowerCase().includes("jquery")
+	);
+};
+const isControlCall = (node: ESTree.Node) => {
+	const sFunctionName = "control";
+	return (
+		node.type === Syntax.CallExpression &&
+		node.callee.type === Syntax.MemberExpression &&
+		node.callee.property.type === Syntax.Identifier &&
+		node.callee.property.name === sFunctionName
```
Comment (tobiasso85) [inline src/tasks/helpers/finders/JQueryControlCallFinder.ts:18]: can be added as "control" inline

File: src/tasks/helpers/finders/JQueryControlCallFinder.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		) {
+			return {configName: sConfigName};
+		} else {
+			return EMPTY_FINDER_RESULT;
+		}
+	}
+}
+
+/**
+ *
```
Comment (tobiasso85) [inline src/tasks/helpers/finders/JQueryControlCallFinder.ts:82]: an example would be nice, what it is all possible to find with the finder

File: src/tasks/helpers/replacers/UI5ElementClosestTo.ts [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+import * as ESTree from "estree";
+import * as recast from "recast";
+import {ASTReplaceable, NodePath} from "ui5-migration";
+
+const builders = recast.types.builders;
+
+type ParamArrayClosestTo = [ESTree.MemberExpression?, ESTree.Literal?];
+
+/**
+ * represents a module which exposes several functions such as
```
Comment (tobiasso85) [inline src/tasks/helpers/replacers/UI5ElementClosestTo.ts:11]: remove this comment and add an appropriate one

File: test/replaceJQueryTest.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
+import {
```
Comment (tobiasso85) [inline test/replaceJQueryTest.ts:1]: hm I would see this as part of addMinissingDependencies test

and the test files could go into a subfolder

File: defaultConfig/addMissingDependencies.config.json [hunk 5]
```diff
@@ -9,14 +9,22 @@
...
 				"newModulePath": "sap/ui/dom/jquery/control",
 				"replacer": "AddComment",
 				"commentText": " jQuery Plugin \"control\"",
 				"finder": "JQueryFunctionExtensionFinderWithDependencyCheck",
 				"extender": "AddUnusedImportWithComment",
 				"version": "^1.58.0"
 			},
+			"*.control@1.106.0": {
+				"newModulePath": "sap/ui/core/Element",
+				"newVariableName": "UI5Element2",
```
Comment (tobiasso85) [inline defaultConfig/addMissingDependencies.config.json:22]: should be changed to "UI5Element"

**Review by tobiasso85 — CHANGES_REQUESTED** [review]

File: defaultConfig/addMissingDependencies.config.json (hunk 5 above)
Comment (stopcoder) [reply defaultConfig/addMissingDependencies.config.json:22]: done

File: src/tasks/helpers/finders/JQueryControlCallFinder.ts (hunk 2 above)
Comment (stopcoder) [reply src/tasks/helpers/finders/JQueryControlCallFinder.ts:82]: done

File: src/tasks/helpers/replacers/UI5ElementClosestTo.ts (hunk 3 above)
Comment (stopcoder) [reply src/tasks/helpers/replacers/UI5ElementClosestTo.ts:11]: done

File: test/replaceJQueryTest.ts (hunk 4 above)
Comment (stopcoder) [reply test/replaceJQueryTest.ts:1]: done

**Review by tobiasso85 — APPROVED** [review]
LGTM

**Pull request merged**